cmd/main.go

```diff
@@ -144,6 +137,28 @@ func parseConfig() (Config, error) {
 			retval.ignoreNamespacesRegex = append(retval.ignoreNamespacesRegex, regex)
 		}
 	}
+
+	switch strings.ToLower(retval.ctrlConfig.LogLevel) {
+	case "debug":
+		logLevel = zapcore.DebugLevel
+	case "warn":
+		logLevel = zapcore.WarnLevel
+	case "error":
+		logLevel = zapcore.ErrorLevel
+	default:
+		logLevel = zapcore.InfoLevel
```

Comment:
IMO, since this is a new configuration value, we should fail if provided a non-sensical level value instead of silently switching to "info".